ci/envs/311-dev.yaml

```diff
@@ -7,6 +7,7 @@ dependencies:
   # required
   - numpy
   - pyproj
+  - gdal
```

Comment:
does adding gdal here do anything, given that pyogrio is installed with pip? I would have thought the GDAL bundled in the wheel would be used regardless of whether we install with conda (but my understanding might be wrong)

Reply:
It is installed with pip but from source, not from a wheel. We need to compile it on installation for which we need GDAL available.